### PR TITLE
[#35] crud, logging complete test remain

### PR DESCRIPTION
**Pull request by RujanRanjitkar**
Done CRUD operation in springboot
Done logging using properties file
Made custom exception

### CODE REVIEW

File: rujanranjitkar/springboot/springbootCRUD/src/main/java/com/example/springbootcrud/model/Product.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            generator = "product_sequence"
+    )
+
+    @Column(name = "product_id")
+    private long productId;
+
+    @Column(name = "product_name")
+    private String productName;
+
+    @Column(name = "price")
```
Comment (preetbista) [inline rujanranjitkar/springboot/springbootCRUD/src/main/java/com/example/springbootcrud/model/Product.java:33]: not necessary in single word.

File: rujanranjitkar/springboot/springbootCRUD/src/main/java/com/example/springbootcrud/service/ProductServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    public List<Product> getAllProducts() {
+        logger.info("Selecting all products from database: " + productRepo.findAllProducts().size());
+        return productRepo.findAllProducts();
+    }
+
+    @Override
+    public void addNewProduct(ProductDto productDto) {
+        Product product;
+        if (productDto.getProductId() != null)
+            product = productRepo.findProductById(productDto.getProductId()).orElse(new Product());
```
Comment (preetbista) [inline rujanranjitkar/springboot/springbootCRUD/src/main/java/com/example/springbootcrud/service/ProductServiceImpl.java:36]: you don't have to check id while adding

**Review by preetbista — COMMENTED** [review]
resolve the above comments

**Review by GyalbuSherpa5 — APPROVED** [review]
ok

**Review by Crack-way — APPROVED** [review]
Ok.

**Review by anujmani — APPROVED** [review]
ok

**Review by sumanpaudel333 — APPROVED** [review]
ok

**Review by SatyaRajAwasth1 — APPROVED** [review]

**Pull request merged**